### PR TITLE
[locoex-customop] Improve README

### PR DESCRIPTION
**Pull request by pfusik**
This corrects the grammar.

ONE-DCO-1.0-Signed-off-by: Piotr Fusik <p.fusik@samsung.com>

### CODE REVIEW

**Comment by pfusik** [comment]
Splitting #12695 to multiple PRs as requested by @seanshpark

**Comment by seanshpark** [comment]
> Commit body word check: 4 words
The body of your commit does not satisfy this repository requirements.
The body needs to contain at least 5 words.

File: compiler/locoex-customop/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # locoex
 
-_locoex_ is an extention of loco. Classes with `COp` prefix enables *Custom Operation*.
+_locoex_ is an extension of loco. Classes with the `COp` prefix enable *Custom Operation*.
 In this version, a *custom operation* means one of the following:
 
-1. an op that is supported by Tensorflow but not supported both by the moco and the onert
```
Comment (pfusik) [inline compiler/locoex-customop/README.md:6]: "not supported both" is still unclear. What if only one of them supports the op?

File: compiler/locoex-customop/README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-1. an op that is not supported by Tensorflow, moco, and the onert
+1. an op that is supported by Tensorflow but not supported both by moco and onert
+1. an op that is not supported by Tensorflow, moco or onert
```
Comment (pfusik) [inline compiler/locoex-customop/README.md:7]: That should probably be `2.`

File: compiler/locoex-customop/README.md (hunk 1 above)
Comment (seanshpark) [reply compiler/locoex-customop/README.md:6]: I think `both` is not that important. either can benefit from this module.

Comment (pfusik) [reply compiler/locoex-customop/README.md:6]: Do moco and onert support same ops? If not, it's either:

    1. an op that is supported by Tensorflow but not supported by moco or onert

or

    1. an op that is supported by Tensorflow but not supported by both moco and onert

and these mean different things.

Comment (seanshpark) [reply compiler/locoex-customop/README.md:6]: I think meaning of this is CustomOp is for Ops that are not supported from existing Ops (from schema) that we have to support from TensorFlow models.
For this, I think it is the first one.
As I understand, the existing writing using `and` is used for (both moco and onert) can "benefit" from using CustomOp.

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**